assets/component/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,13 @@
+name: component
+channels:
+- conda-forge
+dependencies:
+- python=3.9
+- pip=21.3.1
```

Comment:
probably worth to bump pip to something more recent

Reply:
upgraded to 24.0 version instead of 21.3.1 after compatibility check